src/meshctrl/session.py

```
@@ -479,6 +479,16 @@ async def list_devices(self, details=False, group=None, meshid=None, timeout=Non
             raise exceptions.ServerError(res0["result"])
         if details:
             nodes = json.loads(res0["data"])
+            try:
```

For the future, I don't like putting "try/except/print" type code in a library (; it assumes that the user is viewing the output intently, and otherwise the library will effectively fail silently. Better to throw an exception, then the user can handle and print if they so choose.